Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.UWP/MapRenderer.cs

```diff
@@ -46,13 +59,12 @@ private MapControl NativeMap
             {
                 var mapModel = e.OldElement;
                 MessagingCenter.Unsubscribe<Map, MoveToRegionMessage>(this, "MapMoveToRegion");
-                ((ObservableCollection<Pin>)mapModel.Pins).CollectionChanged -= OnCollectionChanged;
```

Comment:
Yes that should be safe, Pins will now be managed through PinLogic, like the other two projects. rather than in MapRenderer